hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java

```diff
@@ -279,7 +277,6 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           .collect(Collectors.toList());
       omKeyInfo.appendNewBlocks(newLocationList, false);
 
-      omVolumeArgs = getVolumeInfo(omMetadataManager, volumeName);
```

Comment:
All the use of the getVolumeInfo method has been removed, so getVolumeInfo should not be needed, we can [delete it.]( https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java#L620)

Reply:
Done